### PR TITLE
Migrate companies features to cypress

### PR DESCRIPTION
**Pull request by PippoRaimondi**
## Description of change

Migrate all company related features to cypress

## Test instructions

`yarn test:e2e:dit`

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to develop?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

**Comment by PippoRaimondi** [comment]
Keeping the PR small so it's an enjoyable experience to go through it.

File: test/end-to-end/cypress/specs/DIT/collection-spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const selectors = require('../../../../selectors')
+const { assertCollection } = require('../../support/assertions')
+
+const checkCollection = () => {
```
Comment (rafenden) [inline test/end-to-end/cypress/specs/DIT/collection-spec.js]: No need to create this function if it's used only once.

File: test/end-to-end/cypress/specs/DIT/companies-spec.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+const selectors = require('../../../../selectors')
+
+const { createContact } = require('../../support/user-actions')
+
+describe('Advisors', () => {
+  const globalManagerTable = 2
+  const AdviserTable = 3
```
Comment (rafenden) [inline test/end-to-end/cypress/specs/DIT/companies-spec.js]: Should start with lowercase.

**Comment by PippoRaimondi** [comment]
> I know at this stage you are just migrating the tests over but do we want to change the urls from `/companies/0f5216e0-849f-11e6-ae22-56b6b6499611/edit` to something like `/companies/${company.id}/edit` ?

yes, you are absolutely right! This is part of my refactor plan... the only reason I haven't done this yet is because I need to move the fixtures folders within our functional up a level... that means I would need to refactor functionals path and I was hoping to do that in a later stage and on its own PR so it doesn't get confusing... would you be ok with that?

File: test/end-to-end/cypress/specs/DIT/collection-spec.js (hunk 1 above)
Comment (PippoRaimondi) [reply test/end-to-end/cypress/specs/DIT/collection-spec.js]: will fix and push

File: test/end-to-end/cypress/specs/DIT/companies-spec.js (hunk 2 above)
Comment (PippoRaimondi) [reply test/end-to-end/cypress/specs/DIT/companies-spec.js]: will fix and push

**Pull request merged**